deprecated/components/FileUploaderField/index.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
```

Comment:
What is a .stories file please ? :)

Reply:
We use storybook to isolate and showcase Flamingo components https://storybook.js.org/docs/basics/introduction/